env/env.go

```diff
@@ -42,6 +43,17 @@ var envVars map[string]string
 
 var currentEnv = "default"
 
+// GetSpecDir returns the specification directory.
+// It checks whether the environment variable for gauge_specs_dir is set.
+// It returns 'specs' otherwise
+func GetSpecDir() string {
+	var specFromProperties = os.Getenv(SpecsDir)
+	if specFromProperties != "" {
+		return specFromProperties
+	}
+	return "specs"
```

Comment:
Here we can use the constant `common. SpecsDirectoryName`. That would better than hard coding the `specs` string here.

Reply:
agree